dpanonymize/__init__.py

```diff
@@ -140,6 +175,8 @@ def lock_lochness(Lochness: 'Lochness',
             })
         pii_table_loc = phoenix_root.parent / 'pii_convert.csv'
         df.to_csv(pii_table_loc)
+        kwargs['pii_table_loc'] =  pii_table_loc
```

Comment:
This and the following line already exists in PR #14 which should result in conflict. This two PRs should be in tandem: 14-->15-->master

Reply:
Good to know that we can do it this way. Let me try changing the base to #14 -> #15

Reply:
Just changed the #14's base to #15, and merged it. Thanks for letting me know this.